plugin/lsp_settings.vim

```diff
@@ -169,13 +169,14 @@ function! s:vim_lsp_install_server(ft, command) abort
 endfunction
 
 function! s:vim_lsp_settings_suggest(ft) abort
-  if empty(s:vim_lsp_installer(a:ft))
+  let l:entry = empty(s:vim_lsp_installer(a:ft))
```

Comment:
There's a mistake here. Remove empty() this line.

Reply:
Thank you. Fixed.